### PR TITLE
Detail Calico BGP route reflector requirements

### PR DESCRIPTION
**Pull request by Vlaaaaaaad**
As per Lance Robson's( aka @lwr20 ) [message in Slack](https://kubernetes.slack.com/archives/C3QUFP0QM/p1537959279000100?thread_ts=1537951563.000100&cid=C3QUFP0QM), above 50-100 nodes Calico recommends usage of BGP route reflectors:
> Clearly I'm not impartial here, but I can tell you that we test Calico to 5K nodes.
> We recommend BGP route reflectors once you go above ~50-100 nodes (we test up to 100 nodes without RRs).  The reason for RRs is that by default calico sets up BGP connections in a full mesh - clearly the number of connections will scale as the number of nodes squared.  With RRs, the number of BGP connections scales linearly with nodes - but then you have to manage the RRs too.
> The next version of Calico (v3.3 - code is in master already, but not heavily tested yet) will bring a feature to make RRs much easier to deploy - just annotate a few nodes and they become RRs.

Adding this detail to documentation as I recently got a cluster with such issues.

### CODE REVIEW

File: docs/networking.md [hunk 1]
```diff
@@ -158,7 +158,7 @@ Reference: [Calico 2.1 Release Notes](https://www.projectcalico.org/project-cali
 Note that Calico by default, routes between nodes within a subnet are distributed using a full node-to-node BGP mesh.
 Each node automatically sets up a BGP peering with every other node within the same L2 network.
 This full node-to-node mesh per L2 network has its scaling challenges for larger scale deployments.
-BGP route reflectors can be used as a replacement to a full mesh, and is useful for scaling up a cluster.
+BGP route reflectors can be used as a replacement to a full mesh, and is useful for scaling up a cluster. BGP route reflectors are recommended once the number of nodes goes above ~50-100.
 The setup of BGP route reflectors is currently out of the scope of kops.
 
 Read more here: [BGP route reflectors](http://docs.projectcalico.org/latest/usage/routereflector/calico-routereflector)
```
Comment (lwr20) [inline docs/networking.md]: That link is broken - correct URL is: https://docs.projectcalico.org/latest/usage/routereflector

Comment (Vlaaaaaaad) [reply docs/networking.md]: Did not even check that 😆 will fix this. Thank you!

**Comment by justinsb** [comment]
/approve
/lgtm

Thanks @Vlaaaaaaad 

Seems like a fairly big limitation of calico :-(

**Comment by Vlaaaaaaad** [comment]
This is a docs change and has 0 impact. Might be a flaky test? Either way, rerunning.

/test pull-kops-e2e-kubernetes-aws

**Pull request merged**